internal/bminventory/inventory.go

```diff
@@ -567,6 +574,19 @@ func (b *bareMetalInventory) formatNodeIgnitionFile(apiVipDnsname string) ([]byt
 	return buf.Bytes(), nil
 }
 
+func (b *bareMetalInventory) createAndUploadNodeIgnition(ctx context.Context, clusterID *strfmt.UUID, apiVipDnsname string) error {
+	ignitionString, err := b.formatNodeIgnitionFile(apiVipDnsname)
+	if err != nil {
+		return errors.Errorf("Failed to create ignition string for cluster %s", clusterID)
+	}
+	fileName := fmt.Sprintf("%s/%s", clusterID, workerIgnition)
+	err = b.objectHandler.Upload(ctx, ignitionString, fileName)
```

Comment:
log?

Reply:
the log on line 1270 will output the appropriate error

Reply:
and it all goes well? Dont we want to know it happened :)

Reply:
fixed